src/core/MOM_forcing_type.F90

```diff
@@ -729,6 +771,7 @@ subroutine extractFluxes1d(G, GV, US, fluxes, optics, nsw, j, dt, &
     if (.not. do_enthalpy) then
       net_heat(i) = net_heat(i) + (scale * dt * I_Cp_Hconvert * &
                     (fluxes%heat_content_lrunoff(i,j) + fluxes%heat_content_frunoff(i,j) + &
+                     fluxes%heat_content_lrunoff_glc(i,j) + fluxes%heat_content_frunoff_glc(i,j) + &
```

Comment:
This expression would benefit form parentheses specifying the order of arithmetic, but because the previous version of this line used an indeterminate order of arithmetic perhaps we do not need to deal with this before these changes are merged into the main version of MOM6.